guides/react_checkout_guide/10_finalizing_the_order_on_the_backend.md

```diff
@@ -0,0 +1,77 @@
+## Finalizing the order
+
+We will start by adding the functionality to the backend. 
```

Comment:
Add a small description of what is going to be added. Atm the title of this document is the only thing describing what we are about to do